R/autoxgboost.R

```diff
@@ -35,10 +35,10 @@
 #'   It is useful to restrict to fraction to be rather small to speed up the calcuation of the initial design.
 #'   If \code{NULL} the full range defined in \code{par.set} is used.
 #'   Default is \code{c(0.5, 0.55)}.
-#' @param factor.encoder [\code{numeric(1)}]\cr
-#'   Defines how factor variables are handled. Either \code{"impact"} for impact encoding (see \code{\link{makeImpactFeaturesWrapper}}).
-#'   Or \code{"dummy"} for Dummy encoding (see \code{\link[mlr]{createDummyFeatures}}).
-#'   Default is \code{"impact"}.
+#' @param impact.encoding.boundary [\code{integer(1)}]\cr
+#'   Defines the threshold on how factor variables are handled. Factors with more levels than the \code{"impact.encoding.boundary"} get impact encoded (see \code{\link{makeImpactFeaturesWrapper}}) while factor variables with less or equal levels than the \code{"impact.encoding.boundary"} get dummy encoded.
+#'   (see \code{\link[mlr]{createDummyFeatures}}).
```

Comment:
Add line how to encode everything dummy/impact

Reply:
done